### PR TITLE
Add Jellyfin extension

### PR DESCRIPTION
**Pull request by mcarlton00**

### CODE REVIEW

**Comment by kingosticks** [comment]
Is this extension Python 3 *and* Python 2 compatible?  The README only provides Python 2 install instructions and the [minimum Mopidy version is 2.0](https://github.com/jellyfin/mopidy-jellyfin/blob/master/setup.py#L28). The codebase seems to include a lot of Python 2 leftovers (e.g. [here](https://github.com/jellyfin/mopidy-jellyfin/blob/master/mopidy_jellyfin/__init__.py#L1), [here](https://github.com/jellyfin/mopidy-jellyfin/blob/master/mopidy_jellyfin/__init__.py#L28)). The [tox config is old too](https://github.com/jellyfin/mopidy-jellyfin/blob/master/tox.ini#L7). I guess it's up to you if you want to support both versions but is it worth it?

**Comment by mcarlton00** [comment]
The plan is supporting Python 3 only moving forward, I just haven't finished adjusting all the various pieces, so thanks for pointing those out.  One of those is linking the install docs to [our wiki](https://jellyfin.org/docs/general/clients/installing-mopidy.html) instead of hosting them in the repo

File: _ext/jellyfin.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+layout: ext
```
Comment (kingosticks) [inline _ext/jellyfin.md]: None of the others have this, does it do anything?

File: _ext/jellyfin.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+title: Mopidy-Jellyfin
+logo: /media/ext/jellyfin.png
+type: backend
+service: Jellyfin
```
Comment (kingosticks) [inline _ext/jellyfin.md]: This is only used when there's an oauth section so we don't need this.

File: _ext/jellyfin.md (hunk 1 above)
Comment (jodal) [reply _ext/jellyfin.md]: It's applied by default by the rules in `_config.yml`, so it is redundant.

Comment (mcarlton00) [reply _ext/jellyfin.md]: It was in the example in the readme.  If it's not needed I can pull it out

File: _ext/jellyfin.md (hunk 2 above)
Comment (mcarlton00) [reply _ext/jellyfin.md]: I'm assuming you mean just the `service` line?

Comment (kingosticks) [reply _ext/jellyfin.md]: Yep, sorry, should have been clearer.

File: _ext/jellyfin.md (hunk 1 above)
Comment (kingosticks) [reply _ext/jellyfin.md]: Thanks for pointing that out, I've updated the example. Might as well remove it so Jellyfin is consistent with the others.

**Comment by kingosticks** [comment]
Super, thanks!

**Pull request merged**